torchvision/ops/boxes.py

```diff
@@ -133,6 +134,54 @@ def clip_boxes_to_image(boxes: Tensor, size: Tuple[int, int]) -> Tensor:
     return clipped_boxes.reshape(boxes.shape)
 
 
+def box_convert(boxes: Tensor, in_fmt: str, out_fmt: str) -> Tensor:
+    """
+    Converts boxes from given in_fmt to out_fmt.
+
+    Arguments:
+        boxes (Tensor[N, 4]): boxes which will be converted.
+        in_fmt (str): Input format of given boxes.
+        out_fmt (str): Output format of given boxes.
+
+    Returns:
+        boxes (Tensor[N, 4]): Boxes into converted format.
+    """
+    allowed_fmts = ("xyxy", "xywh", "cxcywh")
+    assert in_fmt in allowed_fmts
+    assert out_fmt in allowed_fmts
+
+    if in_fmt == out_fmt:
+        # return boxes.clone()  # to ensure always returning a copy
+        boxes_converted = boxes.clone()
+        return boxes_converted
+
+    else:
```

Comment:
Given that we already returned before, we can remove this `else` case and save one level of identation

Reply:
I added this in current PR, lets leave next one for torchscript.